<project
        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns="http://maven.apache.org/POM/4.0.0">

    <modelVersion>4.0.0</modelVersion>

    <groupId>com.crashnote.module</groupId>
    <artifactId>module-servlet</artifactId>
    <version>0.2</version>
    <packaging>jar</packaging>

    <!-- ==== Module is NOT deployed, it is embedded within the released artifacts ==== -->

    <name>Crashnote Servlet Module</name>
    <description>
        Module to catch exceptions from Java servlet apps.
    </description>

    <parent>
        <groupId>com.crashnote</groupId>
        <artifactId>parent-module</artifactId>
        <version>0.2</version>
        <relativePath>../../parents/module</relativePath>
    </parent>

    <dependencies>
        <dependency>
            <groupId>com.crashnote.module</groupId>
            <artifactId>module-logger</artifactId>
            <version>0.2</version>
        </dependency>

        <dependency>
            <groupId>log4j</groupId>
            <artifactId>log4j</artifactId>
        </dependency>

        <dependency>
            <groupId>ch.qos.logback</groupId>
            <artifactId>logback-classic</artifactId>
        </dependency>

        <dependency>
            <groupId>javax.servlet</groupId>
            <artifactId>servlet-api</artifactId>
        </dependency>

        <!-- Test: Servlet Container -->
        <dependency>
            <groupId>org.eclipse.jetty</groupId>
            <artifactId>jetty-webapp</artifactId>
            <version>7.5.1.v20110908</version>
            <scope>test</scope>
        </dependency>
    </dependencies>

    <build>
        <testSourceDirectory>src/test/scala</testSourceDirectory>
    </build>

    <properties>
        <client-name>cn-servlet</client-name>
    </properties>

</project>
